pyuff.py

```diff
@@ -763,6 +763,8 @@ def _write55(self, fh, dset):
             #             if dset.has_key('r4') and dset.has_key('r5') and dset.has_key('r6'):
             if ('r4' in dset) and ('r5' in dset) and ('r6' in dset):
                 nDataPerNode = 6
+            else:
+                nDataPerNode = 3
```

Comment:
@nilswagner Why is this else required?

Reply:
If you have translational DOFs solely (i.e. no rotational DOFs).

Reply:
if the two lines are missing ...

>>> import pyuff
>>> pyuff.prepare_test_55('nils.unv')
Traceback (most recent call last):
  File "/home/wagner/git/pyuff/pyuff.py", line 780, in _write55
    dset['spec_data_type'], dataType, nDataPerNode))
UnboundLocalError: local variable 'nDataPerNode' referenced before assignment